maint/patches/optional/confdb/nv-compiler.patch

```diff
@@ -0,0 +1,61 @@
+--- /home/flyxian/libtool.m4	2020-09-15 21:38:52.847054276 -0500
++++ confdb/libtool.m4	2020-09-16 00:55:57.103309019 -0500
+@@ -4445,8 +4445,8 @@
+ 	    _LT_TAGVAR(lt_prog_compiler_pic, $1)='-fPIC'
+ 	    _LT_TAGVAR(lt_prog_compiler_static, $1)='-static'
+ 	    ;;
+-	  pgCC* | pgcpp*)
+-	    # Portland Group C++ compiler
++	  pgCC* | pgcpp* | pgc++* | nvc++* )
++	    # NVIDIA HPC C++ Compiler
+ 	    _LT_TAGVAR(lt_prog_compiler_wl, $1)='-Wl,'
+ 	    _LT_TAGVAR(lt_prog_compiler_pic, $1)='-fpic'
+ 	    _LT_TAGVAR(lt_prog_compiler_static, $1)='-Bstatic'
+@@ -4788,9 +4788,8 @@
+ 	_LT_TAGVAR(lt_prog_compiler_pic, $1)='-fPIC'
+ 	_LT_TAGVAR(lt_prog_compiler_static, $1)='-static'
+ 	;;
+-      pgcc* | pgf77* | pgf90* | pgf95* | pgfortran*)
+-        # Portland Group compilers (*not* the Pentium gcc compiler,
+-	# which looks to be a dead project)
++      pgcc* | pgf77* | pgf90* | pgf95* | pgfortran* | nvc | nvfortran* )
++        # NVIDIA HPC Compilers
+ 	_LT_TAGVAR(lt_prog_compiler_wl, $1)='-Wl,'
+ 	_LT_TAGVAR(lt_prog_compiler_pic, $1)='-fpic'
+ 	_LT_TAGVAR(lt_prog_compiler_static, $1)='-Bstatic'
+@@ -4834,7 +4833,7 @@
+ 	  _LT_TAGVAR(lt_prog_compiler_pic, $1)='-fPIC'
+ 	  _LT_TAGVAR(lt_prog_compiler_static, $1)='-static'
+ 	  ;;
+-	*Portland\ Group*)
++	*Portland\ Group* | *NVIDIA\ Compilers* | *PGI\ Compilers*)
+ 	  _LT_TAGVAR(lt_prog_compiler_wl, $1)='-Wl,'
+ 	  _LT_TAGVAR(lt_prog_compiler_pic, $1)='-fpic'
+ 	  _LT_TAGVAR(lt_prog_compiler_static, $1)='-Bstatic'
+@@ -5262,12 +5261,12 @@
+ 	tmp_addflag=' $pic_flag'
+ 	tmp_sharedflag='-shared'
+ 	case $cc_basename,$host_cpu in
+-        pgcc*)				# Portland Group C compiler
++        pgcc* | nvc )				# NVIDIA HPC C++ compiler
```

Comment:
comment should be `NVIDIA HPC C compiler`?

Reply:
True. Let me fix it.